node_modules/oae-core/tenantlandingpage/js/tenantlandingpage.js

```diff
@@ -40,6 +40,18 @@ define(['jquery', 'oae.core'], function($, oae) {
             $.ajax({
                 'url': '/api/tenant/landingPage',
                 'success': function(blocks) {
+                    // Replace all instances of ${tenantDisplayName} in block
```

Comment:
We have something very similar in the `i18n` api as well:

```
            if (translation !== null) {
                // Replace all of the dynamic variables, if provided
                if (variables) {
                    $.each(variables, function(dynamicVariable, dynamicTranslation) {
                        translation = translation.replace('${' + dynamicVariable + '}', dynamicTranslation);
                    });
                }
                return translation;
            }
```

I wonder if it would be worth abstracting it into a simple utility function that replaces variables in a string which can then be used in both places.